src/SkyImages.jl

```diff
@@ -19,7 +19,11 @@ export break_blob_into_tiles, break_image_into_tiles
 export get_local_sources
 export stitch_object_tiles
 
```

Comment:
Do we still need `import Grid`? It shows up in the `REQUIRE` file too. Always glad to have fewer dependencies.